documentation/docs/steps/checkChangeInDevelopment.md

```diff
@@ -6,6 +6,37 @@
 
 * No prerequisites
 
+**Note:** This step is deprecated. Use the [isChangeInDevelopment](isChangeInDevelopment.md) instead.
+
+You can keep most of the step specific configuration parameters in your configuration file `config.yml` untouched. The new step support the old naming convention. However, it is recommended to adjust your parameters to the new steps.
```

Comment:
- supports*

- "the new step" is refers to the isChangeInDevelopment step? In that case I would write "the new [isChangeInDevelopment](isChangeInDevelopment.md) step"

- not sure, but just a suggeston: adjust your parameters to naming convention of the new step.*